.github/workflows/layer0.yml

```diff
@@ -1,19 +1,19 @@
-name: Deploy to Layer0
+name: Deploy to Edgio
 
 on:
   workflow_dispatch:
   push:
 
 jobs:
-  deploy-to-layer0:
+  deploy-to-edgio:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 14
+          node-version: 16
           registry-url: https://npm-proxy.fury.io/moovweb/
       - run: npm ci
-      - run: npm run layer0:deploy -- --token=$LAYER0_DEPLOY_TOKEN
+      - run: npm run edgio:deploy -- --token=$EDGIO_DEPLOY_TOKEN
         env:
-          LAYER0_DEPLOY_TOKEN: ${{secrets.LAYER0_DEPLOY_TOKEN}}
+          EDGIO_DEPLOY_TOKEN: ${{secrets.EDGIO_DEPLOY_TOKEN}}
```

Comment:
${{secrets.LAYER0_DEPLOY_TOKEN}} can be used for now, tho ideally we shal l change to EDGIO_DEPLOY_TOKEN

Reply:
changed